WebRoot/js/zimbraMail/core/ZmZimbraMail.js

```diff
@@ -2346,6 +2346,8 @@ function() {
     el = document.getElementById("skin_dropMenu");
     if (el) {
 		this._helpButton = this.getDropMenuOptions(DwtShell.getShell(window), el, adminUrl);
+		ZmZimbraMail.helpButton = this._helpButton;
+		ZmZimbraMail.helpButton.getMenu().setItemVisible("modernClientLink",false);
```

Comment:
Why is the button just changed to invisible rather than removing the code? Please describe more details; requirement, issue, and/or design.

Reply:
Done